python/eos/plot/config.py

```diff
@@ -41,10 +41,6 @@
 
 matplotlib.rcParams['text.usetex'] = True
 
-matplotlib.rcParams['pgf.preamble'] = [
-    r'\usepackage[hidelinks]{hyperref}',
-    r'\usepackage{amsmath}',
-    r'\usepackage{xcolor}'
-]
+matplotlib.rcParams['pgf.preamble'] = r'\usepackage[hidelinks]{hyperref}\usepackage{amsmath}\usepackage{xcolor}'
```

Comment:
Thanks for the initiative Philip! Can you modify this a little bit? I would prefer to have a multiline python string here, i.e.,

```
matplotlib.rcParams['pgf.preamble'] = r'''
\usepackage{...}
'''
```

with one package per line. This makes it easier to read diffs!